notifications.bs

```diff
@@ -291,6 +291,14 @@ removed from the <a>list of notifications</a>.
 <p>User agents should display <a>persistent notifications</a> in a platform's
 "notification center" (if available).
 
+<p>User agents should only expose properties on the {{Notification}} object when
```

Comment:
I think it's better to not say "on the Notification object" as they are more likely on the prototype object and maybe elsewhere.

We should probably also mention that dictionary members are likewise not to be supported (this impacts whether valueOf is run, for instance).